### DISCUSSION TITLE
How to implement multiple HTML in the same folder without creating multiple folders ?

### DISCUSSION THREAD

**Discussion by sanjinhub** [vikejs/vike#1518] [Discussion]
<img width="281" alt="image" src="https://github.com/vikejs/vike/assets/53846155/8c7d77ac-62d4-4e04-a4ad-72bab94eaaba">

**Comment by brillout** [comment]
Currently you can't, but it's on the radar. (I've a couple of ideas.)

**Comment by sanjinhub** [comment]
Thanks
